File: MODUL 6/PRAK6/PRAK603-2410817210025-Helga.py
a, b = map(int, input().split())
if(a != b):
    print("Jumlah tidak sama")
else:
    barisa = list(map(int, input().split()))
    barisb = list(map(int, input().split()))
    for i in range (a):
        print(barisa[i] * barisb[i], end = ' ')

File: MODUL 6/PRAK 6/PRAK603-2410817210025-Helga.c
#include <stdio.h>

int main(){
    int a, b;
    scanf("%d %d", &a, &b);
    if(a != b){
        printf("Jumlah tidak sama");
    }
    else{
        int baris1[a]; 
        int baris2[b];
        for(int i = 0; i < a; i++){
            scanf("%d", &baris1[i]);
        }
        for(int i = 0; i < b; i++){
            scanf("%d", &baris2[i]); 
        }
        for(int i = 0; i < a; i++){
            int n = baris1[i] * baris2[i];
            printf("%d ", n);
        }
    }
}